frontend/ui/widget/filechooser.lua

```diff
@@ -232,6 +232,9 @@ end
 
 function FileChooser:init()
     self.path_items = {}
+    if lfs.attributes(self.path, "mode") ~= "directory" then
+        self.path = lfs.currentdir()
```

Comment:
That's KOReader's Lua code working directory. Isn't it better to you the "home directory" (or the default book location that we use whe no home set) as the fallback dir ? (not sure when that happens, when a file get removed and if have a dead self.path if I read well the other issue).